documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,29 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
```

Comment:
```suggestion
{project-short} validates the migration plan, and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} validates the plan again.
```
You must improve your use of basic grammar:
* `{project-short} validates the migration plan, and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error.` is two independent clauses. Use a comma before "and" if it merges two independent clauses. 
* Poor punctuation - missing period at the end of the sentence. 

FYI - @raoradhika

Reply:
Fixed.

Reply:
@RichardHoch it seems like the changes were not posted